### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by kylematthewjohnson**
Made style changes to the frontend, making it more fleshed out. Added the filtering menu, which is not yet incorporated with backend.

### CODE REVIEW

File: src/main/java/frontend/src/Layout.js [hunk 1]
```diff
@@ -193,6 +193,8 @@ const useStyles = makeStyles((theme) => ({
         // backgroundImage: `url(https://thepaintpeople.com/wp-content/uploads/2015/09/prepare-bare-wood-staining.jpg)`,
         // backgroundImage: `url(https://www.larchwoodcanada.com/wp-content/uploads/larchwood-classic-cutting-board-large_7652.jpg)`,
         backgroundImage: `url(https://www.hillwoodproducts.com/wp-content/uploads/2015/03/wood1.jpg)`,
+        // backgroundColor: '#bae0f7',
+        // backgroundColor: 'blue',
```
Comment (TripleATeam) [inline src/main/java/frontend/src/Layout.js]: Why just comment them out? Are they going to be readded eventually?

File: src/main/java/frontend/src/MenuBar.js [hunk 2]
```diff
@@ -2,22 +2,28 @@ import React from 'react';
...
         flexGrow: 1,
     },
-    menuButton: {
-        marginRight: theme.spacing(2),
-    },
-    title: {
-        flexGrow: 1,
-        textAlign: "start",
-    },
+    // menuButton: {
```
Comment (TripleATeam) [inline src/main/java/frontend/src/MenuBar.js]: Happens in a few places - I hope it's just a temporary thing, but should probably be cleaned up.

**Comment by jkalm-1950533** [comment]
Looks good

**Review by TripleATeam — APPROVED** [review]
Comments are a little interesting, other than that looks good! Up to you to merge if you want to keep the comments.

**Review by jkalm-1950533 — APPROVED** [review]
Looks good

**Pull request merged**